interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,451 @@
+                    <list>socks4 socks5 http exec</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>socks4</format>
+                    <description>socks4 Proxy</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>socks5</format>
+                    <description>socks4 Proxy</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>http</format>
+                    <description>http Proxy</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>exec</format>
+                    <description>Executes the given command which should set up the outgoing connection. The environment variables NAME, NODE, REMOTEADDRES and REMOTEPORT are available</description>
+                  </valueHelp>
+                  <constraint>
+                    <regex>(socks4|socks5|http|exec)</regex>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="address">
+                <properties>
+                  <help>Proxy Address</help>
+                  <constraint>
+                    <validator name="ip-host"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="port">
+                <properties>
+                  <help>Proxy Port</help>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-65535"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="username">
+                <properties>
+                  <help>Proxy Username</help>
+                </properties>
+              </leafNode>
+              <leafNode name="password">
+                <properties>
+                  <help>Proxy Password</help>
+                </properties>
+              </leafNode>
+              <leafNode name="exec">
+                <properties>
+                  <help>Executes the given command which should set up the outgoing connection. The environment variables NAME, NODE, REMOTEADDRES and REMOTEPORT are available</help>
+                </properties>
+              </leafNode>
+            </children>
+          </node>
+          <leafNode name="replay-window">
+            <properties>
+              <help>ReplayWindow(Bytes,default:16)</help>
+            </properties>
+            <defaultValue>16</defaultValue>
+          </leafNode>
+          <leafNode name="strict-subnets">
+            <properties>
+              <help>When this option is enabled tinc will only use Subnet statements which are present in the host config files in the local /etc/tinc/netname/hosts/ directory. Subnets learned via connections to other nodes and which are not present in the local host config files are ignored</help>
```

Comment:
I think we should avoid overly long help strings. "Ignore connections from unknown networks" would suffice, a detailed description should be in docs.vyos.io

Reply:
done